hotsos/core/plugins/kubernetes.py

```diff
@@ -70,24 +65,17 @@ def bind_interfaces(self):
 
     @cached_property
     def pods(self):
-        if self._pods:
-            return self._pods
-
         _pods = []
```

Comment:
nit: could probably just be 'pods' and below 'containers', guessing used _ to match private name?

Reply:
wont that trigger a pylint error as it overrides a global name?

Reply:
hmm no didnt